src/main/java/org/datadog/jmxfetch/AppConfig.java

```diff
@@ -106,7 +106,8 @@
     @Parameter(
             names = {"--reporter", "-r"},
             description =
-                    "Reporter to use: should be either \"statsd:[STATSD_PORT]\", "
+                    "Reporter to use: should be either \"statsd:[STATSD_HOST][STATSD_PORT]\", "
+                     + "\"statsd:[STATSD_SOCKET_PATH]\", "
```

Comment:
Just to be a little more explicit
```suggestion
                     + "\"statsd:[STATSD_UNIX_SOCKET_PATH]\", "
```